code/game/machinery/recycle_vendor.dm

```diff
@@ -249,12 +270,17 @@
 			var/amount = currentitemmaterials[materialtype]
 			totalmaterials[materialtype] += amount
 		currentitemsvalue *= 0.2 // 20% fee to recycle
+		currentitemsvalue = CEILING(currentitemsvalue, 1) // good luck keeping your money here, although this is still cheaper than rounding twice
 		moneyinput -= currentitemsvalue
 		var/datum/transaction/T = new(currentitemsvalue, "", "Recycling fee for [currentitem.name]", src)
 		T.apply_to(silo.my_account)
+		silo.taxdebt += currentitemsvalue/32
```

Comment:
in the case of sale via Recycler, taxes are one thirty-second of the sale value, which is recompense for the ability to the AME to be bailed out.